test/internal/polyfills/BigInt.js

```diff
@@ -0,0 +1,5 @@
+import BigInt from 'big-integer';
```

Comment:
Let's create a `test/shared/BigInt.js` file instead and drop this polyfill usage. (remove also from package.json/lock)


The file content will look like this:

```js
export default typeof BigInt === 'function' ? BigInt : undefined;
```

**Explanation**: I'd rather we test on real native values or not test at all. We always avoided to testing our predicate function on polyfills. IMHO it doesn't really make sense to test on behavior that is not native.

Reply:
Ok, I was trying to work around the fact the library was being tested on earlier versions of node. But I see your point.